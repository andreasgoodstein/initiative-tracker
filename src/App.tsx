import React from 'react';
import { Provider } from 'react-redux';

import Navigator from './navigator';

import { configureStore } from './store';

const store = configureStore();

export default () => (
  <Provider store={store}>
    <Navigator />
  </Provider>
);
