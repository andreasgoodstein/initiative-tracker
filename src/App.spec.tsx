import { shallow } from "enzyme";
import React from "react";

import App from "./App";

describe("the Actor component", () => {
    it("renders", () => {
        const wrapper = shallow(<App></App>);

        expect(wrapper).not.toBeNull();
        expect(wrapper).toMatchSnapshot();
    });
});
