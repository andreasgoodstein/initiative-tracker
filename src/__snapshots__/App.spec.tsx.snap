// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the Actor component renders 1`] = `
<Provider
  store={
    Object {
      "dispatch": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
      Symbol(observable): [Function],
    }
  }
>
  <Connect(Component) />
</Provider>
`;
